.github/workflows/docker_build.yml

```diff
@@ -1,7 +1,10 @@
 name: Publish Docker
-on: [push]
+on: 
+  push: {}
+  schedule:
+    - cron:  '0 0 1 * *'
```

Comment:
Let's do it at least once per week (if not daily), TF versions are released really quickly

Reply:
Changed to daily builds